usr/share/rear/output/USB/Linux-i386/100_create_efiboot.sh

```diff
@@ -88,7 +89,8 @@ EOF
 
                 # Create bootloader, this overwrite BOOTX64.efi copied in previous step ...
                 # Fail if BOOTX64.efi can't be created
-                ${GRUB_MKIMAGE} -o ${EFI_DST}/BOOTX64.efi -p ${EFI_DIR} -O x86_64-efi linux part_gpt ext2 normal gfxterm gfxterm_background gfxterm_menu test all_video loadenv fat
+                # Slackware grub doesnt have gfxterm_background or gfxterm_menu...
+                ${GRUB_MKIMAGE} -o ${EFI_DST}/BOOTX64.efi -p ${EFI_DIR} -O x86_64-efi linux part_gpt ext2 normal gfxterm gfxmenu test all_video loadenv fat
```

Comment:
I think here a test for Slackware and conditional code execution is needed
like you did it in your usr/share/rear/pack/Linux-i386/300_copy_kernel.sh
<pre>
# add slackware test on top to prevent error because
# slackware grub doesnt have gfxterm_background or gfxterm_menu...
if [ -f /etc/slackware-version ] ; then
    ${GRUB_MKIMAGE} -o ${EFI_DST}/BOOTX64.efi -p ${EFI_DIR} -O x86_64-efi linux part_gpt ext2 normal gfxterm gfxmenu test all_video loadenv fat
else
    ${GRUB_MKIMAGE} -o ${EFI_DST}/BOOTX64.efi -p ${EFI_DIR} -O x86_64-efi linux part_gpt ext2 normal gfxterm gfxterm_background gfxterm_menu test all_video loadenv fat
fi
</pre>
or perhaps even cleaner by using a menaingful variable for the GRUB modules like
<pre>
grub_modules="x86_64-efi linux part_gpt ext2 normal gfxterm gfxmenu test all_video loadenv fat"
# slackware grub doesnt have gfxterm_background or gfxterm_menu
test -f /etc/slackware-version || grub_modules="$grub_modules gfxterm_background gfxterm_menu"
# Create bootloader, this overwrite BOOTX64.efi copied in previous step ...
# Fail if BOOTX64.efi can't be created
${GRUB_MKIMAGE} -o ${EFI_DST}/BOOTX64.efi -p ${EFI_DIR} -O $grub_modules
</pre>

Reply:
Changing modules that are currently used and tested was my only concern. If this code can be re-written so it does not interfere with existing distributions, I have not problems left with this PR.

V.